### PR TITLE
Convert RSS feeds to Atom

### PR DESCRIPTION
**Pull request by ketralnis**
Required by the corresponding r2 change

### CODE REVIEW

**Comment by ketralnis** [comment]
:eyeglasses: @spladug @dellis23

**Comment by spladug** [comment]
Commit messages/diff structure is important in several ways to the review process:

- it's critical to how more complicated pieces of code get deployed
- it can divide up discrete changes so reviewers can focus on one at a time with less overhead
- they end up as the permanent record for posterity to figure out what we were thinking/doing, so it's important they themselves get review

and so we like to have the commits in :shipit: shape before the pull request goes out for review (this is not so necessary when it's a sketch that you're getting early feedback on like the first drafts of the permacache stuff).  Could you rework all these "x"es please?

File: reddit_liveupdate/templates/liveupdate.xml [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
-<item>
-    <guid isPermaLink="true">https://${get_domain(cname=False, subreddit=False)}/live/${c.liveupdate_event._id}/updates/${thing._id}</guid>
-    <pubDate>${thing._date.strftime('%a, %d %b %Y %H:%M:%S %z')}</pubDate>
-    <description>
-        ${spaceCompress(safemarkdown(thing.body))}
-    </description>
-</item>
+<%namespace name="utils" file="utils.xml"/>
+<entry>
+    %if not thing.author.deleted:
```
Comment (spladug) [inline reddit_liveupdate/templates/liveupdate.xml]: what's going on with this super long line?

Comment (13steinj) [inline reddit_liveupdate/templates/liveupdate.xml]: Stupid question here: Isn't it supposed to be `thing.author._deleted`?

Comment (ketralnis) [inline reddit_liveupdate/templates/liveupdate.xml]: No, I thought so at first too. author here is a wrapper Template object
that only has some of the attributes of an actual Account object and
renames deleted without the underscore

Comment (ketralnis) [inline reddit_liveupdate/templates/liveupdate.xml]: It's a long constructed string literal. But it can be factored out above for sure

**Comment by spladug** [comment]
general :fish: but a couple things might change based on the r2 PR (/u/ and /r/, author refactoring)

**Comment by ketralnis** [comment]
:eyeglasses: @spladug final pass

**Pull request merged**